src/Monitor/Monitor.Test/SessionRecords/Microsoft.Azure.Commands.Insights.Test.ScenarioTests.AutoscaleTests/TestAddAzureRmAutoscaleSetting.json

```diff
@@ -4,15 +4,15 @@
+            "RequestBody": "{ \"properties\": { \"profiles\": [ { \"name\": \"adios\", \"capacity\": { \"minimum\": \"1\", \"maximum\": \"10\", \"default\": \"1\" }, \"rules\": [ { \"metricTrigger\": { \"metricName\": \"Requests\", \"metricResourceUri\": \"/subscriptions/b67f7fec-69fc-4974-9099-a26bd6ffeda3/resourceGroups/TestingMetricsScaleSet/providers/Microsoft.Compute/virtualMachineScaleSets/testingsc\", \"timeGrain\": \"PT1M\", \"statistic\": \"Average\", \"timeWindow\": \"PT1M\", \"timeAggregation\": \"Average\", \"operator\": \"GreaterThan\", \"threshold\": 10.0 }, \"scaleAction\": { \"direction\": \"Increase\", \"value\": \"1\", \"cooldown\": \"PT5M\", \"type\": \"ChangeCount\" } }, { \"metricTrigger\": { \"metricName\": \"Requests\", \"metricResourceUri\": \"/subscriptions/b67f7fec-69fc-4974-9099-a26bd6ffeda3/resourceGroups/TestingMetricsScaleSet/providers/Microsoft.Compute/virtualMachineScaleSets/testingsc\", \"timeGrain\": \"PT2M\", \"statistic\": \"Average\", \"timeWindow\": \"PT1M\", \"timeAggregation\": \"Average\", \"operator\": \"GreaterThan\", \"threshold\": 15.0 }, \"scaleAction\": { \"direction\": \"Decrease\", \"value\": \"2\", \"cooldown\": \"PT6M\", \"type\": \"ChangeCount\" } } ], \"fixedDate\": { \"timeZone\": \"GMT\", \"start\": \"2015-03-05T14:00:00Z\", \"end\": \"2015-03-05T14:30:00Z\" } }, { \"name\": \"saludos\", \"capacity\": { \"minimum\": \"1\", \"maximum\": \"10\", \"default\": \"1\" }, \"rules\": [ { \"metricTrigger\": { \"metricName\": \"Requests\", \"metricResourceUri\": \"/subscriptions/b67f7fec-69fc-4974-9099-a26bd6ffeda3/resourceGroups/TestingMetricsScaleSet/providers/Microsoft.Compute/virtualMachineScaleSets/testingsc\", \"timeGrain\": \"PT1M\", \"statistic\": \"Average\", \"timeWindow\": \"PT1M\", \"timeAggregation\": \"Average\", \"operator\": \"GreaterThan\", \"threshold\": 10.0 }, \"scaleAction\": { \"direction\": \"Increase\", \"value\": \"1\", \"cooldown\": \"PT5M\", \"type\": \"ChangeCount\" } }, { \"metricTrigger\": { \"metricName\": \"Requests\", \"metricResourceUri\": \"/subscriptions/b67f7fec-69fc-4974-9099-a26bd6ffeda3/resourceGroups/TestingMetricsScaleSet/providers/Microsoft.Compute/virtualMachineScaleSets/testingsc\", \"timeGrain\": \"PT2M\", \"statistic\": \"Average\", \"timeWindow\": \"PT1M\", \"timeAggregation\": \"Average\", \"operator\": \"GreaterThan\", \"threshold\": 15.0 }, \"scaleAction\": { \"direction\": \"Decrease\", \"value\": \"2\", \"cooldown\": \"PT6M\", \"type\": \"ChangeCount\" } } ], \"recurrence\": { \"frequency\": \"Week\", \"schedule\": { \"timeZone\": \"UTC\", \"days\": [ \"1\" ], \"hours\": [ 5 ], \"minutes\": [ 15 ] } } } ], \"enabled\": true, \"name\": \"MySetting\", \"targetResourceUri\": \"/subscriptions/b67f7fec-69fc-4974-9099-a26bd6ffeda3/resourceGroups/TestingMetricsScaleSet/providers/Microsoft.Compute/virtualMachineScaleSets/testingsc\" }, \"name\": \"MySetting\", \"location\": \"East US\", \"tags\": {}}",
```

Comment:
Did you update these files individually?  You need to re-record them if you make a change like this.

Reply:
I change manually. What else do I need to change ?

Reply:
These tests need to be re-recorded.  You cannot edit these files manually